client/my-sites/site-settings/settings-newsletter/newsletter-categories-section/newsletter-categories-section.tsx

```diff
@@ -1,50 +1,31 @@
 import { Card } from '@automattic/components';
-import { useIsEnglishLocale } from '@automattic/i18n-utils';
-import { useI18n } from '@wordpress/react-i18n';
 import classNames from 'classnames';
 import { useTranslate } from 'i18n-calypso';
 import React from 'react';
 import TermTreeSelector from 'calypso/blocks/term-tree-selector';
 import FormSettingExplanation from 'calypso/components/forms/form-setting-explanation';
-import SettingsSectionHeader from 'calypso/my-sites/site-settings/settings-section-header';
 import NewsletterCategoriesToggle from './newsletter-categories-toggle';
 import './style.scss';
 
 type NewsletterCategoriesSectionProps = {
 	disabled?: boolean;
 	handleToggle: ( field: string ) => ( value: boolean ) => void;
-	handleSubmitForm: () => void;
 	newsletterCategoriesEnabled?: boolean;
 	newsletterCategoryIds: number[];
 	updateFields: ( fields: { [ key: string ]: unknown } ) => void;
-	isSavingSettings: boolean;
 };
 
 const NewsletterCategoriesSection = ( {
 	disabled,
 	handleToggle,
 	newsletterCategoriesEnabled,
 	newsletterCategoryIds,
-	handleSubmitForm,
 	updateFields,
-	isSavingSettings,
 }: NewsletterCategoriesSectionProps ) => {
 	const translate = useTranslate();
-	const { hasTranslation } = useI18n();
-	const isEnglishLocale = useIsEnglishLocale();
 
 	return (
 		<>
-			{ /* @ts-expect-error SettingsSectionHeader is not typed and is causing errors */ }
-			<SettingsSectionHeader
-				id="newsletter-categories-settings"
-				title={ translate( 'Newsletter categories settings' ) }
-				showButton
-				onButtonClick={ handleSubmitForm }
-				disabled={ disabled }
-				isSaving={ isSavingSettings }
-			/>
-
```

Comment:
Moved settings header from this file to `main.jsx` for consistency with other sections which also now have their header in the main file.